.travis.yml

```diff
@@ -4,10 +4,15 @@ os:
   - osx
 julia:
   - 0.6
+  - 1.0
   - nightly
 matrix:
   allow_failures:
     - julia: nightly
+branches:
+  only:
+    - master
+    - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags
```

Comment:
This is to avoid triggering a build both for the PR branch and the PR itself. The version tags thing is useful for if this package ever gets Documenter documentation.